src/encoded/static/components/item-pages/FileView.js

```diff
@@ -95,17 +95,31 @@ export default class FileView extends WorkflowRunTracingView {
 
 function FileViewOverview (props) {
     const { context, windowWidth, width, schemas, href } = props;
-    const experimentSetUrls = expFxn.experimentSetsFromFile(context, 'ids');
+    const experimentSets = expFxn.experimentSetsFromFile(context, 'list');
+    const searchHref =
+        experimentSets && experimentSets.length > 0 ?
+            '/search/?type=ExperimentSet&accession=' + _.pluck(experimentSets, 'accession').join('&accession=')
+            : null;
+    const expSetTableProps = {
+        searchHref: searchHref,
+        facets: null,
+        defaultOpenIndices: [0],
+        title: <ExperimentSetsTableTabViewTitle />,
+        externalSearchLinkVisible: false,
```

Comment:
This is set to false everywhere, is there anywhere we do want to show the external link?

Reply:
Will push a couple of minor updates; we don't need to pass around `externalSearchLinkVisible` so much I think.

Reply:
Ah am guessing is for ExperimentTypeView (where is 'true' implicitly (?))